compiler/src/dotty/tools/dotc/core/Contexts.scala

```diff
@@ -480,7 +483,22 @@ object Contexts {
 
     /** A new context that summarizes an import statement */
     def importContext(imp: Import[?], sym: Symbol): FreshContext =
-       fresh.setImportInfo(ImportInfo(sym, imp.selectors, imp.expr))
+      fresh.setImportInfo(ImportInfo(sym, imp.selectors, imp.expr))
+
+    def scalaRelease: ScalaRelease =
+      val releaseName = base.settings.YscalaRelease.value
+      if releaseName.nonEmpty then ScalaRelease.parse(releaseName).get else ScalaRelease.latest
+
+    def tastyVersion: TastyVersion =
+      import math.Ordered.orderingToOrdered
+      val latestRelease = ScalaRelease.latest
+      val specifiedRelease = scalaRelease
+      if ((specifiedRelease.majorVersion, specifiedRelease.minorVersion) < (latestRelease.majorVersion, latestRelease.minorVersion)) then
```

Comment:
I saw that `ScalaRelease` implements `Ordered`
```suggestion
      if specifiedRelease < latestRelease then
```